src/components/Login.tsx

```diff
@@ -0,0 +1,175 @@
+"use client";
+
+import { useState } from "react";
+import TextField from "@mui/material/TextField";
+import Checkbox from "@mui/material/Checkbox";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import InputAdornment from '@mui/material/InputAdornment';
+import IconButton from '@mui/material/IconButton';
+
+import VisibilityOffIcon from "@mui/icons-material/VisibilityOff";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+
+import Image from "next/image"
+import logo1 from "../../public/logo1.png"
+
+export default function LoginForm() {
+  const [showPassword, setShowPassword] = useState(false);
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [displayError, setDisplayError] = useState(false);
+
+  const CustomInputProps = {
+    endAdornment: (
+      <InputAdornment position="end">
+        <IconButton>
+          {password != "" &&
+            (showPassword ? (
+              <VisibilityIcon
+                sx={{ color: "#138D8A" }}
+                onClick={() => setShowPassword(false)}
+              />
+            ) : (
+              <VisibilityOffIcon
+                sx={{ color: "#138D8A" }}
+                onClick={() => setShowPassword(true)}
+              />
+            ))}
+        </IconButton>
+      </InputAdornment>
+    ),
+  };
+
+  return (
+    <div className="flex flex-col items-center" style={{ minHeight: "100vh" }}>
```

Comment:
Could you guys use Tailwind styling when possible? I know it might not work with MUI components but for regular components like divs let's keep it consistent!